app/src/main/AndroidManifest.xml

```diff
@@ -8,12 +8,19 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.Papilio">
-
+        <activity
+            android:name=".create_activity"
```

Comment:
What is this activity doing here? The file is completely empty and it's not following camel case spelling convention for classes. I believe it should be removed